docs/api.md

```diff
@@ -94,6 +94,8 @@ As you repeatedly call `resolveConfig`, the file system structure will be cached
 }
 ```
 
+The promise will be rejected if the type of `filePath` is not `string`.
```

Comment:
I think it would be better to add type annotations to all docs. This just _happens_ to reject the promise due to `path.relative` throwing an error.

Reply:
sounds good